car_control.py

```diff
@@ -1,137 +1,128 @@
-#!/usr/bin/env python3.7
+#!/usr/bin/env python3
 print("Loading Python modules for car_control. Please be patient.")
-import rcpy, datetime, time, math
+import rcpy, datetime, time, math, socket, json
 #import pygame
-from rcpy.servo import servo1
-from rcpy.servo import servo3
+# from rcpy.servo import servo1
+# from rcpy.servo import servo3
```

Comment:
Maybe we could put this delta into some kind of global configuration?

Reply:
Do you have any suggestions on a clean and simple way of implementing such an on/off configuration of the car_control? Currently the only clean/simple way that we are able to think of involved a duplicate car_control.py that has the debug comment changes, which we would swap import lines for.